### ISSUE TITLE
No accessible name provided for Skip and Limit Spin Buttons: A11y_DocumentDB for VS Code Extension_View Query Insights_Name

### ISSUE THREAD

**Issue by AnKushSingh05** [microsoft/vscode-documentdb#377]
### GitHub ### Tags
#A11yTCS; #A11yMAS; #A11ySev2; #DesktopApp; #FTP; #Win32; #DocumentDB for VS Code Extension; #SH_DocumentDB_Win32_NOV2025; #WCAG4.1.2;#Name Role Value;

### Environment Details:
Application Name: DocumentDB for VS Code Extension
Version 0.6.0
OS: Windows 11[Version 23H2(OS Build 22631.3593)]

### Repro Steps:
Open VS code
Install the DocumentDB for VS Code Extension and open it
Navigate to the DocumentDB Connections area.
Click the New Connection action.
Select Connection String as the connection method.
Paste the connection string when prompted.
Choose to authenticate with username/password and enter the credentials
Confirm to create the connection.
Locate the desired cluster in the DocumentDB Connections list.
Expand the cluster entry to initiate the connection.
Expand one of the databases under the cluster.
Expand a collection within that database.
Select the Documents item for that collection.
Collection View will open
Observe the issues No Accessible Name Provided for Skip and Limit Spin Buttons
### Actual Result:
No Accessible Name Provided for Skip and Limit Spin Buttons
Similar issue is also observed for 

### Expected Result:
An accessible name should be provided to the spin buttons.

### User Impact:
Users relying on assistive technologies, such as screen readers, cannot perceive the purpose or functionality of the spin buttons for "Skip" and "Limit." Without a programmatically determinable name, these controls are inaccessible to users with visual impairments.

### Attachment

<img width="1280" height="720" alt="Image" src="https://github.com/user-attachments/assets/7f52ab70-9ed3-4d5b-b986-8c714fde7e6d" />

<img width="1280" height="720" alt="Image" src="https://github.com/user-attachments/assets/4acbd84f-00e1-490d-aff7-b56a27b46a8e" />
